### ISSUE TITLE
Support for adding SVG images to cards

### ISSUE THREAD

**Issue by user1823** [ankidroid/Anki-Android#9606]
I often need to add SVG images to my cards. However, the option provided in AnkiDroid to add images seemingly doesn't support SVG.
So, I have to first go to the file manager to the move the SVG file (say, example.svg) to the collection media folder and then type (&lt;img src="example.svg"&gt;) in the desired part of the card. This process is very tedious.
I think there should be a shorter way. Possibly, the option provided in AnkiDroid to add images may be slightly modified to support adding SVGs.

**Comment by mikehardy** [comment]
Interesting. If the Android WebView supports it, I don't see why not? @user1823 you confirm that when you go to use cards that have .svg file references they render just fine?

For anyone that implements this:
- the file picker has a list of media types that it accepts I believe? Check in the Multimediacard package area to find where we open the file picker and see if we can just add .svg to the list of types we accept
- we will likely need to add an "isMutable(type of image)" method and avoid attempting to crop or rotate/compress when `!isMutable`

that may be it?

**Comment by mikehardy** [comment]
Since WebView defines what we support, perhaps the preview could use a WebView to render the images there?

**Comment by david-allison-1** [comment]
Yep! As long as we ensure that normal images go through the compress stage

**Comment by david-allison-1** [comment]
sounds good to me
